tests/routes/mocks.js

```diff
@@ -144,20 +144,31 @@ const anotherCaller = {
   },
 };
 
-const withHelpline1 = {
+const another1 = {
   ...contact1,
   form: { ...contact1.form, childInformation: anotherChild },
   helpline: 'Helpline 1',
 };
 
-const withHelpline2 = {
+const another2 = {
   ...contact2,
-  form: { ...contact2.form, callerInformation: anotherCaller, childInformation: anotherChild },
+  form: {
+    ...contact2.form,
+    callerInformation: {
+      ...anotherCaller,
+      location: { phone1: '+1 (515) 555-1212', phone2: '+1 (616) 555-1212' },
+    },
+    childInformation: {
+      ...anotherChild,
+      location: { phone1: '+1 (313) 555-1212', phone2: '+1 (414) 555-1212' },
+    },
+  },
   helpline: 'Helpline 2',
+  number: '(212) 555-1212',
```

Comment:
This isn't quite what I mean... the number here is the number we're pulling directly from the Twilio metadata.  It will always be in [E.164](https://en.wikipedia.org/wiki/E.164) format, with the +, the country code and no spaces.  So, `+12125551212`.  What might _not_ be in that format is counselor-entered data.  So what I was looking for in a test more like if `childInformation.location.phone1` is `(313) 555-1212` and then there's a search for `+13135551212`.  Does that make sense?

Reply:
Totally, changing it right away!